### ISSUE TITLE
iOS8 Safari causes absolute position bug

### ISSUE THREAD

**Issue by jaread83** [artberri/sidr#212]
Have tested this on iOS6, 7 and 8. 6 and 7 are totally fine but there is a problem with iOS8 Safari. When opening the SIDR menu at the top of the page, it is fine but then when you scroll down the page and click the SIDR menu the pullout menu is above my viewport slightly. scrolling all the way to the top of the page fixes it but that isn't any good if you are halfway down the page. You can open the SIDR menu and scroll up and down and the pullout menu will go up and down slightly when it should be fixed to the top of the page.
Has anyone else encountered this issue?

**Comment by mischi05** [comment]
You can fix this with CSS "transform"... view code here: 
http://www.michaelloy.de/jquery-sidr-ios-safari-fixed-element/jquery-sidr-ios-safari-fixed-element.html

**Comment by KZeni** [comment]
Seems this issue is also present in iOS 9 (beta).

Opening the Sidr menu on the left pushes the body over, but iOS 9 Safari also scales the site to fit the menu and (now slid-over) site even if you have the viewport set to device-width and disabled user scaling. Meanwhile, opening a Sidr menu on the right doesn't have this problem (likely due to how having elements overflow on the left and/or top doesn't extend the canvas).

Either way, I believe I've found a fix to make it keep the site's scale/viewport the same in both cases.

I've posted a pull request with the quick & simple fix here: https://github.com/artberri/sidr/pull/268
